### ISSUE TITLE
pnpm behavior on global install + save=false

### ISSUE THREAD

**Issue by StreetStrider** [pnpm/pnpm#2261]
### pnpm version: 4
Hello. I've already posted question on gitter related to that issue, but after further investigation I think this worth mentioning in tracker.

Previous discussion:
Strider:
> @zkochan, hello. I've found another strange behavior specific for no-save users. When I'm using `pnpm i -g foo` then `pnpm i -g bar`, after 2nd `foo` will be deleted. I can workaround this with `-S` but this creates package.json for my global packages.
This may be intended behavior, but it is strange for me using this no-save scenario. Both pnpm and npm works normal way for a long time, that you can manually install any package globally without interrupting other global packages. Might be worth to note on this channel.

Zoltan:
> well, foo should not be deleted
there is a global package.json and both foo and bar should be added as dependencies to it

You gave me answer, but today I compared that behavior to npm's behavior.
Basically, I have `save=false` in npmrc, so both npm and pnpm use this settings.
When I install something with `npm i -g foo` it does not teardown anything previously installed.
I think, npm's behavior is correct. It may be formulated like: when using global mode, install any package and treat everything previously installed as manually installed, so keep it too, even if no package.json present or if it's empty.

With pnpm it's not possible to have `save=false` and install bunch of global tools. After installing second tool, 1st will be torn down. So, in any time point only one tool can be installed. To workaround this, no-save users will must always append `-S` to their global installs.

**Comment by StreetStrider** [comment]
Good to know, thanks.
